CHANGELOG.md

```diff
@@ -7,15 +7,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v0.6.0] - 2021-06-24
+### Added
+- Use `async-std` instead of `smol` for async tasks. ([#301](https://github.com/paritytech/substrate-archive/commit/f2a4b408123c5e64be04e70a890b1354475d812d))
+- Additional tests for missing storage & pagination query. ([#254](https://github.com/paritytech/substrate-archive/commit/36d955d379b1fdfb0ff063dce394d8a4d6430323))
+- Storage Chain Support ([#292](https://github.com/paritytech/substrate-archive/commit/70ea92187ed68aee9c9cc8de757b1b86bb7ab408))
+- `v0.9.5`, `v0.9.6` wasm tracing runtimes
+
 ### Changed
-- Use `async-std` instead of `smol` for async tasks. ([#301](https://github.com/paritytech/substrate-archive/pull/301))
+- **BREAKING**: PostgreSQL `_background_tasks` table stores data as `jsonb` rather than bytea. Upgrading should happen
+  automatically when running v0.6.0, but you will not be able to revert to any previous version of archive.
+  ([#254](https://github.com/paritytech/substrate-archive/commit/36d955d379b1fdfb0ff063dce394d8a4d6430323))
+- When restoring storage, blocks are paginated down to `max_block_load` parameter. This should cut memory usage down
+where lots of blocks were missing from storage at one time. ([#254](https://github.com/paritytech/substrate-archive/commit/36d955d379b1fdfb0ff063dce394d8a4d6430323))
+- Update dependencies to match runtime `0.9.6`.
```

Comment:
Aren't the runtime versions like `9060` or some such? Or do you mean client release?

Reply:
Yeah, I meant client here